### ISSUE TITLE
Why this project silent since May ?

### ISSUE THREAD

**Issue by meepeek** [onixjs/core#65]
The web said that beta access was on May but I have got no update. Cannot reach you by email either. I really interesting in your work.

**Comment by roccolucatallarita** [comment]
Hi Jonathan, how are you? We all love this project, can you post some update, please? If you need some help, please ask... We can contribute.

**Comment by ixshel** [comment]
> The web said that beta access was on May but I have got no update. Cannot reach you by email either. I really interesting in your work.

@meepeek we are currently working on that, as we also have our jobs and this is a side project that we love is being a hard time to continue working, but I can say is just a matter of days so we will have this beta working in the next days, thanks.

**Comment by ixshel** [comment]
> Hi Jonathan, how are you? We all love this project, can you post some update, please? If you need some help, please ask... We can contribute.

@RLTmultimedia hey guys we are more than glad that you like it, as I mentioned in a comment before we are extremely near to get this beta out, so just wait a couple of days

**Comment by jonathan-casarrubias** [comment]
Hi guys, I really appreciate your interest on this project.

I want to apologize because I had some hard personal issues that I needed to go through back on May, but the project is not dead and actually is incredible near to be launched to the beta testers.

The only pending portion before we send you invites is to actually fix a couple of dashboard issues from the SSO system.

Our commitment is to fix those UI issues and then we will be in a point to send you the invites so you can register and use the SSO with OnixJS.

Talking about contribution I also would really appreaciate anyone digging into the core and finding improvements, having a community to push the development would also make this project stronger.

I hope to keep seeing you around and if you already have registered to the closed beta, please be patient to receive your invitation within the next following days.

Cheers
Jon

**Comment by jmls** [comment]
urgh - not sure if we registered for beta or not - we've been waiting for the beta to surface and were getting concerned about the lack of communication. One thing that's sure to make people wary is no commits and no chat on a project ;)

**Comment by jonathan-casarrubias** [comment]
@jmls after the core was released on beta, all the efforts moved to the SSO, the thing is that the SSO System is not open source and it's not available on github so you are not able to see commits.

The latests commits on this repo were sent after finding issues while building the SSO,  but certainly being our own users we are pretty sure that the core is working pretty stable now.

Once we release the SSO in beta we will continue adding new features to this repo.

And just to clarify, the beta registration is not to get access to the framework, which you already have, the registration is for you to get free access to the SSO and have authentication and authorization not just for the OnixJS Framework but you will be able to use the service even if you use other backend technologies.

I apologize again for the lack of communication and we will be adding more communication channels in the following days.

Cheers
Jon

**Comment by jmls** [comment]
ok, so to clarify - the "core" is onixjs, but there hasn't really been much change since march ? That kinda surprises me, as it would indicate that the core is both stable and feature-complete .. or is there a boat-load of code in the SSO that will make it into core once you get past beta ?

**Comment by jonathan-casarrubias** [comment]
Sure I can clarify that.

The core which is located on this repo is completely functional to start creating APIs and Clients with no problem.

Using our own core framework, we built our first project on top -as you would create one- we built a SSO (Single Sign On) service running an OpenID Connect protocol.

Said that a limitation you would have at this moment is that the core does not provide authentication mechanism, though it does provide authorization mechanisms through ACLs.

The SSO Registration for Beta Access, is for you to have a robust and standarized authentication service, there you will be able to create organizations, groups and add users to those groups, then you will be able to link those groups with your ACLs groups.

Lets say you don't care about the SSO, well your limitation at this point is that you would need to create a register/login mechanism.

Said that, after we release the SSO Service, we will also add a basic HTTP authentication mechanism within the core, so you would be able to chose either use the basic authentication mechanism or use the robust SSO OIDC service.

There are other features that will be added into the core after the SSO is out

Does this sort of clarify your questions?

**Comment by jmls** [comment]
it somewhat does :) is there a roadmap for the features to be added ? For example, is there a plan to get schematics (or something similar) in place to generate the api calls / auth / models / websockets / client / server stuff instead of having to churn out shitloads of boiler-plate code ?

Will dig into the core over the next few days to have a gander ;)

**Comment by jonathan-casarrubias** [comment]
@jmls yes absolutely, now that we have verified that the core works pretty neat by creating our own project on top -the sso-, we will start making more robust the experience by providing tools, to explore the API, generate code etc.. Things that are not really related to the core functionality but to improve  and speed up the development experience.

Really, any improvements, ideas and contributions you guys might have, we will be pleased to hear about those

**Comment by mikkoh85** [comment]
I think your projects are the best back end tools I have used so far. Creating a Angular front end with Loopback 3 was a breeze with Sdk, and I used FireLoop to build a fully working two player online Chess game just in 4 evenings for a job interview! The interviewer COO of quite a big company here in Oulu, Finland asked me to estimate how long it would take me to build such a game and I said two days. He said he doesn't believe me and I didn't get the job. I got a little angry of that kind of "know it all" comment and I made the game during that work week just to show them. Here's the game, haven't tried if it still works:
https://gitlab.com/mikkoh85/chess

I Wish I could contribute, but time is a factor and I also have my day job. But I could maybe do some testing at least?

**Comment by ptesar-xyndata** [comment]
Hi guys, loopback 4 is in GA state, so I'm thinking to use loopback instead of onix for my next project, as I see last commit in July 2018 :(

**Comment by jgwinner** [comment]
The website is gone? How is the project doing?

**Comment by jgwinner** [comment]
Pin drop ... :(

**Comment by souvrard** [comment]
Hi,
Is this project dead ? website down, no comments here .....

**Comment by jgwinner** [comment]
Yup ... website still down.

I get the "We're too busy getting it up and ready" idea, but the website being down is a bad sign.
